client/src/store/index.js

```diff
@@ -108,6 +109,9 @@ export const mutations = {
   setLoading(state, loading) {
     state.loading = loading;
   },
+  setStyleOptions(state, styleOptions) {
```

Comment:
Apart from this, you'll also need to save the values in Chrome's storage, left a comment on your PR about it.